node-admin/src/test/java/com/yahoo/vespa/hosted/node/admin/integrationTests/DockerTester.java

```diff
@@ -75,8 +74,7 @@
         ipAddresses.addAddress(HOST_HOSTNAME.value(), "f000::");
         for (int i = 1; i < 4; i++) ipAddresses.addAddress("host" + i + ".test.yahoo.com", "f000::" + i);
 
-        ProcessExecuter processExecuter = mock(ProcessExecuter.class);
-        uncheck(() -> when(processExecuter.exec(any(String[].class))).thenReturn(new Pair<>(0, "")));
+        TerminalImpl terminal = new TerminalImpl(command -> new TestChildProcess2(0, ""));
```

Comment:
Why not use TestTerminal?

Reply:
This is basically "integration test" for node-admin: it runs the entire node-admin, starts multiple containers which run regular ticks. There is no way to list all the expected commands here...

Reply:
Yes, TestTerminal needs a method that returns a given exit code & response forever. If it's non-trivial it's OK to leave as-is.